ci/docs/build.sh

```diff
@@ -40,6 +40,7 @@ conda list --show-channel-urls
 #libcudf Doxygen build
 gpuci_logger "Build libcudf docs..."
 cd $PROJECT_WORKSPACE/cpp/doxygen
+wget "https://raw.githubusercontent.com/rapidsai/docs/gh-pages/api/librmm/${BRANCH_VERSION}/rmm.tag" || echo "Failed to download rmm Doxygen tag"
```

Comment:
rmm and cudf may diverge in version numbers. Check with @harrism

Reply:
@karthikeyann If this is true, this should be communicated with OPS ASAP. That could affect a lot more than just doc builds.

Reply:
There are no current plans to diverge. RMM and cuDF are released on the same schedule.